src/_pytest/python.py

```diff
@@ -109,7 +109,7 @@ def pytest_addoption(parser):
     group.addoption(
         "--import-mode",
         default="prepend",
-        choices=["prepend", "append"],
+        choices=["prepend", "append", "importlib"],
         dest="importmode",
         help="prepend/append to sys.path when importing test modules, "
```

Comment:
I think this help string should be updated

Reply:
Yeah.
But initially this was just meant to allow for using this easily already (i.e. experimental).